src/applications/personalization/profile360/components/PaymentInformationEditModal.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import ErrorableTextInput from '@department-of-veterans-affairs/formation-react/ErrorableTextInput';
+import ErrorableSelect from '@department-of-veterans-affairs/formation-react/ErrorableSelect';
+
+const ACCOUNT_TYPES_OPTIONS = {
+  checking: 'Checking',
+  savings: 'Savings',
+};
+
+class PaymentInformationEditModal extends React.Component {
+  static propTypes = {
+    onClose: PropTypes.func.isRequired,
+    onSubmit: PropTypes.func.isRequired,
+    setPaymentInformationUiState: PropTypes.func.isRequired,
+    paymentInformationUiState: PropTypes.object,
+  };
+
+  static defaultEditModalFields = {
+    financialInstitutionRoutingNumber: {
+      field: {
+        value: '',
+        dirty: false,
+      },
+    },
+    accountNumber: {
+      field: {
+        value: '',
+        dirty: false,
+      },
+    },
+    accountType: {
+      options: Object.values(ACCOUNT_TYPES_OPTIONS),
+      value: {
+        value: ACCOUNT_TYPES_OPTIONS.checking,
+        dirty: false,
+      },
+    },
+  };
+
+  componentDidMount() {
+    this.setEditedState(PaymentInformationEditModal.defaultEditModalFields);
+  }
+
+  onClose = () => {
+    this.props.onClose();
+    this.setEditedState(PaymentInformationEditModal.defaultEditModalFields);
+  };
```

Comment:
Here ,the `defaultEditModalFields` are added into the store under the `vaProfile.paymentInformationUiState.editModalFields` to initialize this component's props.